frontend/server/src/main/java/org/pytorch/serve/ModelServer.java

```diff
@@ -83,6 +83,11 @@ public static void main(String[] args) {
             ConfigManager.init(arguments);
             ConfigManager configManager = ConfigManager.getInstance();
             PluginsManager.getInstance().initialize();
+            Map<String, ModelServerEndpoint> plugins =
+                    PluginsManager.getInstance().getManagementEndpoints();
+            if (plugins.containsKey("token")) {
+                configManager.setupTokenClass();
+            }
```

Comment:
Can you move this plugin check in PluginManager?

Reply:
Noted